test/test_files/caver_expected_data/starting.json

```diff
@@ -692,7 +692,7 @@
             "entrances": {},
             "hints": {
                 "0200": {
-                    "text": "Rumour has it, an expansion can be found in Plantation.",
+                    "text": "Rumour has it, an Energy Transfer Module can be found in Plantation.",
```

Comment:
hmmmmmm?

Reply:
this is a bugfix! for reasons I do not comprehend, it was previously possible for a Nothing item to be hinted as an expansion. I've changed it so that Nothings are only ever referred to by name. the name being *wrong* in other games is an existing bug unrelated to this fix